src/envoy/http/metadata_exchange/config.cc

```diff
@@ -106,7 +114,7 @@ Http::FilterHeadersStatus PluginContext::onResponseHeaders() {
       !upstream_metadata_value->view().empty()) {
     removeResponseHeader(ExchangeMetadataHeader);
     auto upstream_metadata_bytes =
-        Base64::decode(upstream_metadata_value->toString());
+        Base64::decodeWithoutPadding(upstream_metadata_value->toString());
```

Comment:
I suspect we can use upstream_metadata_value->view()
I remeber there is decode api which accept string view. So you could save another string construction.